ksql-engine/src/main/java/io/confluent/ksql/function/udf/datetime/DateToString.java

```diff
@@ -39,9 +42,21 @@
   @Udf(description = "Converts an integer representing days since epoch to a string"
       + " using the given format pattern. The format pattern should be in the format"
       + " expected by java.time.format.DateTimeFormatter")
-  public String dateToString(final Integer daysSinceEpoch, final String formatPattern) {
-    final DateTimeFormatter formatter = formatters.getUnchecked(formatPattern);
-    return LocalDate.ofEpochDay(daysSinceEpoch).format(formatter);
+  public String dateToString(
+      @UdfParameter(value = "epochDays",
+          description = "The Epoch Day to convert,"
+              + " based on the epoch 1970-01-01") final Integer epochDays,
```

Comment:
maybe use `int` rather than `Integer` as it'll throw an NPE on null anyway.

Reply:
done